### PR TITLE
Updated version for eth-brownie.

### PR DESCRIPTION
**Pull request by mariacarmina**
Fixes # .

Changes proposed in this PR:

- updated brownie to 1.19.2
- inject POA middleware for polygon test

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "enforce-typing==1.0.0.post1",
     "json-sempai==0.4.0",
     "eciespy",
-    "eth-brownie==1.19.1",
+    "eth-brownie==1.19.2",
```
Comment (trentmc) [inline setup.py:36]: "eth-brownie==1.19.2" -> "eth-brownie"

File: setup.py [hunk 2]
```diff
@@ -63,7 +63,7 @@
     "watchdog",
     "flake8==5.0.4",
     "isort==5.10.1",
-    "black==22.6.0",  # need to keep this up to date to brownie
+    "black==22.10.0",  # need to keep this up to date to brownie
```
Comment (trentmc) [inline setup.py:66]: "black==22.10.0" ->  "black"

**Review by trentmc — CHANGES_REQUESTED** [review]
When a user installs ocean.py, they likely also have other py libraries. If ocean.py forces to a specific version, it may make it impossible to install because of version conflicts.

This is why's it's super-important to not force a specific version on users.

File: tests/integration/remote/test_polygon.py [hunk 3]
```diff
@@ -2,8 +2,10 @@
 # Copyright 2022 Ocean Protocol Foundation
 # SPDX-License-Identifier: Apache-2.0
 #
+import brownie
 import pytest
 from brownie.network import accounts
+from web3.middleware import geth_poa_middleware
```
Comment (trentmc) [inline tests/integration/remote/test_polygon.py:8]: These changes do not belong in this PR. It's a different problem. Create the github issue, create the branch, create the PR, get it approved, merge it. But separately from this one.

**Review by trentmc — CHANGES_REQUESTED** [review]
As commented: the changes wrt  polygon should be in a separate issue / branch / PR.

File: tests/integration/remote/test_polygon.py (hunk 3 above)
Comment (trentmc) [reply tests/integration/remote/test_polygon.py:8]: And, I think the github issue may already exist: https://github.com/oceanprotocol/ocean.py/issues/1187

Comment (mariacarmina) [reply tests/integration/remote/test_polygon.py]: Created #1188
